src/components/views/loginPage.jsx

```diff
@@ -0,0 +1,196 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { PropTypes } from 'prop-types';
+import { loginRequestAction } from '../../actions/loginActions';
+import validateSignIn from '../../utilities/validateInput';
+import AlertBox from '../containers/alerts/AlertBox';
+import '../../../public/styles/auth.scss';
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.focusInput = React.createRef();
```

Comment:
instead of having this in a constructor you can use 
`focusInput = React.createRef();` and then call it using `this.focusInput`
No need for a constructor

Reply:
Okay @seyi-adeleke. Thanks.